realm/migrate.gno

```diff
@@ -0,0 +1,85 @@
+package chess
+
+func checkMigrated() {
+	if migrated {
```

Comment:
```go
if !migrated {return}
...
```

Reply:
Do you mean to handle the situation where the realm has not migrated yet? If that is the case, since `migrated` is a bool, the only stateful change that will happen is if it is `true`. Why check otherwise?